### DISCUSSION TITLE
Help with adding user

### DISCUSSION THREAD

**Discussion by astumpf1** [subdavis/kobo-book-downloader#89] [Q&A]
I'm on MacOS 12.6 Monterey

I've run the user add command, and got to the last step of copying the code to bring back to the terminal. Nothing happens after hitting Enter after copying the code. I've tried pasting instead of hitting enter, with the same results. I can't get past this step.
<img width="813" alt="Screen Shot 2022-09-28 at 3 11 36 PM" src="https://user-images.githubusercontent.com/114618299/192879196-49b14e70-ec4c-4a89-8f6f-ca5be081bd12.png">

**Comment by astumpf1** [comment]
Also, no matter what I do, I can't see to install/update to 9.2. I always seem to have 8.0.

**Reply by puffenuff** [reply]
Ya me too. Did you find an answer?

**Comment by subdavis** [comment]
Authentication has changed to an easier method, please try 0.10.1
